gunrock/oprtr/1D_oprtr/for_all.cuh

```diff
@@ -255,7 +260,7 @@ cudaError_t mgpu_ForAll(const util::MultiGpuContext& mgpuContext,
 
   // launch ForAll kernel for each gpu (on its own thread)
   std::vector<std::thread> threads;
-  threads.reserve(mgpuContext.getGpuCount());
+  // threads.reserve(mgpuContext.getGpuCount());
```

Comment:
This is fine to get rid off, but generally nice to reserve space for exactly the number of items needed (if you happen to know ahead of time). Not an issue for the number of threads we're working with, but for a larger number of items the allocator will end up allocating 50-60% more memory than the vector needs (to avoid reallocating and copying).

Reply:
Oops, forgot to uncomment that. I was testing `resize` vs. `reserve`, `reserve` was faster.